src/components/Group/GroupFeed/GroupFeed.jsx

```diff
@@ -15,17 +16,14 @@ const GroupFeed = ({ groupId, leaderName }) => {
 		(state) => state.post,
 	);
 
-	const [optionMenuOpenedFeedIndex, setOptionMenuOpenedFeedIndex] =
-		useState(null);
-
 	const [isLoading, setIsLoading] = useState(true);
 
-	const groupPostFetching = async () => {
-		await dispatch(getGroupAllPosts(groupId));
-		setIsLoading(false);
-	};
-
 	useEffect(() => {
+		const groupPostFetching = async () => {
+			await dispatch(getGroupAllPosts(groupId));
+			setIsLoading(false);
+		};
+
 		groupPostFetching();
```

Comment:
제가 최근에 효율적인 코드 작성 면에서 발견한 방식이 있는데, 즉시 실행 함수를 사용하는 겁니다! 어치파 이러한 `useEffect` 내부 함수 선언은 비동기 로직을 위해 감싸주는 건데, 사용도 line 27에서 한 번 하기 때문에 다음과 같이 하면 좋습니다.

```suggestion
		(async () => {
			await dispatch(getGroupAllPosts(groupId));
			setIsLoading(false);
		})();
```

제가 작성했던 쪽에 있는 `useEffect` 코드에도 발견될 때마다 수정할 예정입니다.